src/WebJobs.Script.WebHost/Diagnostics/Extensions/ScriptHostServiceSecurityExtension.cs

```diff
@@ -14,22 +14,22 @@ public static class ScriptHostServiceSecurityExtension
             LoggerMessage.Define<string, string>(
                 LogLevel.Error,
                 new EventId(600, nameof(BlobStorageSecretRepoError)),
-                "There was an error performing a {operation} operation on the Blob Storage Secret Repository. Please ensure the '{appSettingName}' connection string is valid.");
+                "There was an error performing a {operation} operation on the Blob Storage Secret Repository. Please ensure the '{appSettingName}' connection string is valid. {exception}.");
```

Comment:
Do we need to explicitly include the exception in the format string? I think [it gets passed to the internal Log method](https://github.com/dotnet/runtime/blob/37c5a3e6bf685bb08550baba54a742d21c91ffe7/src/libraries/Microsoft.Extensions.Logging.Abstractions/src/LoggerMessage.cs#L184) automagically.

Reply:
Also, the number of named parameters in message string (3) not matching with the definition (2).

Reply:
From the kusto logs I was looking at, we don't see any exception details

Reply:
@kshyju I doubled checked this and you're right it is included. So I will remove it from the message string :)